### PR TITLE
Better for you to follow the stile guide: Server (#519)

### LINKED ISSUE #519: Better for you to follow the stile guide: Server
## Description:
For the long time we didn't following the style, namely chapter _Names_ rule 2. It's time to change that!
This should remove such warnings:
```log
/home/runner/work/Juniorgram/Juniorgram/Network/Public/Include/Network/Primitives.hpp: In constructor ‘Network::ChannelInfo::ChannelInfo(uint64_t, uint64_t, const string&)’:
/home/runner/work/Juniorgram/Juniorgram/Network/Public/Include/Network/Primitives.hpp:32:9: warning: declaration of ‘channelName’ shadows a member of ‘Network::ChannelInfo’ [-Wshadow]
   32 |         : creatorID(creatorID), channelID(channelID), channelName(channelName)
      |     
```

## Acceptance Criteria:
Follow the rule and rename variables in proper way with underscore.

### PR DESCRIPTION
**Pull request by memorisecodead**
## Style Server: 
1. Fixing shadowEvents.
2. Adding new checking functions in Server class for Message Handler.
3. Updating constructor of Server.
4. Adding new function for accepting connect of Client.
5. Edited comments according to the style guide.
6. New comments added.
7. Edited class variable names according to the style guide.

### CODE REVIEW

File: Server/Server.Core/Server.Core.Static/Server.cpp [hunk 1]
```diff
@@ -29,347 +32,133 @@ void Server::onClientDisconnect(const std::shared_ptr<Connection>& client)
...
 Server::Server(const uint16_t& port)
-    : mAcceptor(mContext, asio::ip::tcp::endpoint(asio::ip::tcp::v4(), port)),
-      mPostgreManager(
-          std::make_unique<DataAccess::PostgreRepositoryManager>(DataAccess::PostgreAdapter::getInstance<DataAccess::PostgreAdapter>()))
+    :_acceptor(_context, asio::ip::tcp::endpoint(asio::ip::tcp::v4(), port)),
+     _IDCounter(10000),
+     _criticalQueueSize(100),
+     _newThreadsCount(std::thread::hardware_concurrency()),
+     _postgreManager
+     (std::make_unique<PostgreRepositoryManager>(PostgreAdapter::getInstance<PostgreAdapter>()))
```
Comment (Orange-hanter) [inline Server/Server.Core/Server.Core.Static/Server.cpp:161]: have some doubt...

File: Server/Server.Core/Server.Core.Static/Server.cpp [hunk 2]
```diff
@@ -384,80 +173,85 @@ void Server::start()
...
-        if (!error)
-        {
-            std::ostringstream out;
-            out << socket.remote_endpoint();
-            Base::Logger::FileLogger::getInstance().log
-            (
-                "[SERVER] New Connection: " + out.str(), 
-                Base::Logger::LogLevel::INFO
-            );
+    _acceptor.async_accept([this](std::error_code error, asio::ip::tcp::socket socket) { acceptingClientConnection(error,socket); });
```
Comment (Orange-hanter) [inline Server/Server.Core/Server.Core.Static/Server.cpp:210]: need investigation

File: Utility/Utility.Public/Include/Utility/Utility.hpp [hunk 3]
```diff
@@ -160,9 +160,9 @@ inline std::string getTimeNow()
 {
     std::string timeStampStr(20, '\0');
 
-    std::time_t t    = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
-    std::tm     time = Utility::safe_localtime(t);
```
Comment (Orange-hanter) [inline Utility/Utility.Public/Include/Utility/Utility.hpp:164]: bring to merge conflict

File: Server/Server.Core/Server.Core.Static/Server.cpp [hunk 4]
```diff
@@ -384,80 +173,85 @@ void Server::start()
...
-            /// @todo This function always return true
-            if (onClientConnect(newConnection))
-            {
-                mConnectionsPointers.push_back(std::move(newConnection));
+        auto newConnection = std::make_shared<Connection>(Connection::OwnerType::SERVER, _context, std::move(socket), _incomingMessagesQueue);
 
-                mConnectionsPointers.back()->connectToClient(mIDCounter++);
+        if (onClientConnect(newConnection))
+        {
+            _connectionsPointers.push_back(std::move(newConnection));
```
Comment (Orange-hanter) [inline Server/Server.Core/Server.Core.Static/Server.cpp:229]: Change to emplace_back

File: Server/Server.Core/Server.Core.Static/Server.cpp [hunk 5]
```diff
@@ -384,80 +173,85 @@ void Server::start()
...
+            _connectionsPointers.back()->connectToClient(++_IDCounter);
```
Comment (Orange-hanter) [inline Server/Server.Core/Server.Core.Static/Server.cpp:230]: Investigate how this change could affect the performance

**Review by Orange-hanter — CHANGES_REQUESTED** [review]

File: Server/Server.Core/Server.Core.Static/Server.cpp (hunk 2 above)
Comment (memorisecodead) [reply Server/Server.Core/Server.Core.Static/Server.cpp:210]: > 
Below there is a method that gets the parameters for the lambda expression to work correctly. 
It was made in order to reduce the code in the block and sonar would not be perturbed.

File: Server/Server.Core/Server.Core.Static/Server.cpp (hunk 1 above)
Comment (memorisecodead) [reply Server/Server.Core/Server.Core.Static/Server.cpp:161]: In any case, the fields of the class will be initialized in the fields of

Comment (memorisecodead) [reply Server/Server.Core/Server.Core.Static/Server.cpp:161]: > have some doubt...

In any case, the fields of the class will be initialized in the fields of

**Pull request closed without merging**